app/src/main/java/tech/pauly/findapet/discover/DiscoverViewModel.java

```diff
@@ -52,8 +65,26 @@ public AnimalListAdapter getListAdapter() {
         return listAdapter;
     }
 
-    @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)
-    public void loadList() {
+    @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
```

Comment:
This is moved to onResume but only called once because the fragment is not listening to the EventBus at the point in time of onCreate, so we have to do it later.